packages/api/src/utils/logger.ts

```diff
@@ -34,11 +34,9 @@ export class CustomTypeOrmLogger
   }
 
   logQuery(query: string, parameters?: any[], queryRunner?: QueryRunner) {
-    this.logger.info(
-      `query: ${query} -- PARAMETERS: ${super.stringifyParams(
-        parameters || []
-      )}`
-    )
```

Comment:
@jacksonh i changed this line to make sql parameters part of the log entry object so we can reduce the size of parameters accordingly